lms/templates/instructor/instructor_dashboard_2/instructor_dashboard_2.html

```diff
@@ -104,7 +104,7 @@
         <section class="instructor-dashboard-content-2" id="instructor-dashboard-content">
           <h2 class="hd hd-2 instructor-dashboard-title">${_("Instructor Dashboard")}</h2>
           <div class="wrap-instructor-info studio-view">
-            %if studio_url:
+            %if studio_url and not sections[0].get('access').get('staff') or request.user.is_staff:
```

Comment:
Could you please clarify what is expected in the statement?

Reply:
The link will be shown only if the user has is_staff or has 'admin' access and a studio URL exists. And the link will not be displayed if the user has the role “staff” or if he is a student.

Reply:
Now statement works in a little bit different way, without studio_url, and with request.user.is_staff the statement returns True which looks a little bit odd.

Reply:
Done